### PR TITLE
[REACTOS] Fix some spellings

### PR DESCRIPTION
**Pull request by ace-dent**
## Purpose

Fixes mistakes for en-GB locale.
Fix spelling of `unassociate` across source.
Make `read-only`  and `double-click` hyphenation consistent across source.

## Proposed changes

Text has been changed.

### CODE REVIEW

File: dll/3rdparty/libtirpc/man/rpc.3t [hunk 1]
```diff
@@ -222,7 +222,7 @@ struct svc_req {
     rpcvers_t    rq_vers;    /* service protocol version */
     rpcproc_t    rq_proc;    /* the desired procedure */
     struct opaque_auth    rq_cred;    /* raw creds from the wire */
-    caddr_t    rq_clntcred;    /* read only cooked cred */
+    caddr_t    rq_clntcred;    /* read-only cooked cred */
```
Comment (binarymaster) [inline dll/3rdparty/libtirpc/man/rpc.3t:225]: This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/3rd+Party+Files.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l97

File: dll/win32/comctl32/listview.c [hunk 2]
```diff
@@ -9584,7 +9584,7 @@ static BOOL LISTVIEW_DrawTrackLine(const LISTVIEW_INFO *infoPtr)
 /***
  * DESCRIPTION:
  * Called when an edit control should be displayed. This function is called after
- * we are sure that there was a single click - not a double click (this is a TIMERPROC).
+ * we are sure that there was a single click - not a double-click (this is a TIMERPROC).
  *
  * PARAMETER(S):
  * [I] hwnd : Handle to the listview
```
Comment (binarymaster) [inline dll/win32/comctl32/listview.c:9590]: This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l59

File: dll/win32/comdlg32/colordlg.c [hunk 3]
```diff
@@ -373,7 +373,7 @@ static BOOL CC_MouseCheckColorGraph( HWND hDlg, int dlgitem, int *hori, int *ver
 }
 /***********************************************************************
  *                  CC_MouseCheckResultWindow                 [internal]
- *                  test if double click one of the result colors
+ *                  test if double-click one of the result colors
```
Comment (binarymaster) [inline dll/win32/comdlg32/colordlg.c:376]: This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l60

File: dll/win32/msi/dialog.c [hunk 4]
```diff
@@ -3094,7 +3094,7 @@ static UINT msi_dialog_directory_list( msi_dialog *dialog, MSIRECORD *rec )
     prop = MSI_RecordGetString( rec, 9 );
     control->property = msi_dialog_dup_property( dialog, prop, FALSE );
 
-    /* double click to activate an item in the list */
+    /* double-click to activate an item in the list */
```
Comment (binarymaster) [inline dll/win32/msi/dialog.c:3097]: This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l117

File: dll/win32/oleacc/lang/oleacc_De.rc [hunk 5]
```diff
@@ -97,7 +97,7 @@ STRINGTABLE
     IDS_STATE_PRESSED           "pressed"
     IDS_STATE_CHECKED           "checked"
     IDS_STATE_MIXED             "mixed"
-    IDS_STATE_READONLY          "read only"
+    IDS_STATE_READONLY          "read-only"
```
Comment (binarymaster) [inline dll/win32/oleacc/lang/oleacc_De.rc:100]: This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l151

File: modules/rosapps/applications/net/tsclient/rdesktop/uiports/qtewin.cpp [hunk 6]
```diff
@@ -922,7 +922,7 @@ void QMyMainWindow::MemuClicked(int MenuID)
     rdp_send_input(0, RDP_INPUT_SCANCODE, RDP_KEYRELEASE, 0x1d, 0); // control
     rdp_send_input(0, RDP_INPUT_SCANCODE, RDP_KEYRELEASE, 0x38, 0); // alt
   }
-  else if (MenuID == 4) // double click
+  else if (MenuID == 4) // double-click
```
Comment (binarymaster) [inline modules/rosapps/applications/net/tsclient/rdesktop/uiports/qtewin.cpp:925]: This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/3rd+Party+Files.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l9

File: sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp [hunk 7]
```diff
@@ -319,7 +319,7 @@ class FxObject {
     union {
         //
         // This field is set when the object is being contructed or before
-        // Commit() and from then on is read only.
+        // Commit() and from then on is read-only.
```
Comment (binarymaster) [inline sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp:322]: This is 3rd-party code, please send the fix upstream.

(cannot find documentation link for it though)

**Review by binarymaster — COMMENTED** [review]
Please avoid touching 3rd-party code, unless it's a sync with upstream. Such changes should be sent to upstream repositories.

See the documentation for winesync and 3rd-party modules.

**Comment by JoachimHenze** [comment]
Sorry, I can also not approve that. As Stasm correctly pointed out, too many parts target 3rd party code, that we do not want to alter. If it was a PR just about the "unassociate"-part in shell32, I would approve that right away.

**Comment by ace-dent** [comment]
I will fix by removing 3rd party code changes. Thanks

**Comment by ace-dent** [comment]
@JoachimHenze - I'm planning on closing this PR and making a couple of simpler ones. 
I'm *excluding* `shell32` as this is [listed](https://github.com/reactos/reactos/blob/365372dc17172d779ded5e808f69446aec77b707/media/doc/WINESYNC.txt#L184) as a WINE import?... Is that correct? Thanks

**Comment by ace-dent** [comment]
Closing. Replaced with #5533, #5534 - that exclude any 3rd party files.

**Pull request closed without merging**